src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
[STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)